concrete/single_pages/dashboard/system/mail/method.php

```diff
@@ -14,6 +14,19 @@
 
     <fieldset>
         <div class="form-group">
+            <?= $form->label('EMAIL_ENABLED', t('Email Sending')) ?>
+            <div class="radio">
+                <label><?= $form->radio('EMAIL_ENABLED', false, strtoupper($config->get('concrete.email.enabled'))) ?><span><?= t('Disabled') ?></span></label>
+            </div>
+            <div class="radio">
+                <label><?= $form->radio('EMAIL_ENABLED', true, strtoupper($config->get('concrete.email.enabled'))) ?><span><?= t('Enabled') ?></span></label>
```

Comment:
Why strtoupper here?

Reply:
Sorry, I just copied from the existing one. Didn't notice that.

Reply:
Removed